powerfactory/powerfactory-converter/src/test/java/com/powsybl/powerfactory/converter/PowerFactoryImporterTest.java

```diff
@@ -44,6 +45,7 @@ void testBase() {
         assertEquals("POWER-FACTORY", importer.getFormat());
         assertTrue(importer.getParameters().isEmpty());
         assertEquals("PowerFactory to IIDM converter", importer.getComment());
+        assertEquals(List.of("json", "dgs"), importer.getSupportedExtensions());
```

Comment:
To be exhaustive, can you add "powsybl-powerfactory-db" as test dependency and "properties" in this list?
Or at least a comment explaining why it is "missing".

Reply:
Done